### PR TITLE
Demo feedback style changes

### PR DESCRIPTION
**Pull request by BelenRemedi**
# Description:

- CSS style for create target, delete target, flash messages changed.
- drop-down for create target area length option and confirm message for delete target added.

### CODE REVIEW

File: app/assets/stylesheets/delete_target.scss [hunk 1]
```diff
@@ -16,3 +21,21 @@
   border-color: black;
   border-radius: 10px;
 }
+
+.button-delete {
+  text-align: center;
+  margin-left: 50px;
+  margin-right: 50px;
+  margin-top: 10px;
```
Comment (vitogit) [inline app/assets/stylesheets/delete_target.scss]: you can use a short version of margin: 10px 50px 0px 50px

File: app/assets/stylesheets/targets.scss [hunk 2]
```diff
@@ -10,39 +9,79 @@
...
 .topic {
   cursor: pointer;
 }
 
-.topic:hover {
-  background-color: grey;
+.button {
+  margin-left: 50px;
+  margin-right: 50px;
+  margin-top: 10px;
```
Comment (vitogit) [inline app/assets/stylesheets/targets.scss]: use margin shorter version

File: app/assets/stylesheets/errors.scss [hunk 3]
```diff
@@ -2,3 +2,8 @@
   color:red;
   font-size: 15px;
 }
+
+.notice {
+  color:green;
```
Comment (JuanfraM) [inline app/assets/stylesheets/errors.scss]: add a space after :

File: app/assets/stylesheets/targets.scss [hunk 4]
```diff
@@ -10,39 +9,79 @@
...
+  width: 70%;
+  margin-top: 3%;
+  display: flex;
+  flex-direction: column;
+
+  .form-input input {
+    border: 0.5px solid black;
+    box-sizing: border-box;
+    padding: 12px 20px;
+    width: 100%;
```
Comment (JuanfraM) [inline app/assets/stylesheets/targets.scss:18]: width: 100%  is necessary?
It's defined below in .form-input

File: app/assets/stylesheets/targets.scss [hunk 5]
```diff
@@ -10,39 +9,79 @@
...
+  }
+}
+
+.form-input {
+  width: 100%;
+  margin: 5px auto;
+}
+
+.area-select {
+  border: 0.5px solid black;
```
Comment (JuanfraM) [inline app/assets/stylesheets/targets.scss]: you can remove the zero, just:
border: .5px solid black;

File: app/assets/stylesheets/main.scss [hunk 6]
```diff
@@ -10,6 +10,11 @@ body {
   font-family: "Open Sans";
 }
 
+a {
+  text-decoration: none;
+  color: #000000;
```
Comment (JuanfraM) [inline app/assets/stylesheets/main.scss]: why not color: black ??

File: app/assets/stylesheets/delete_target.scss [hunk 7]
```diff
@@ -16,3 +21,19 @@
...
   border-radius: 10px;
 }
+
+.button-delete {
+  text-align: center;
+  margin: 10px 50px 0px 50px;
+  background-color: black;
+  border: none;
+  padding: 16px 32px;
+  -webkit-transition-duration: 0.4s; /* Safari */
```
Comment (vitogit) [inline app/assets/stylesheets/delete_target.scss:31]: What is this for?

File: app/assets/stylesheets/index.scss [hunk 8]
```diff
@@ -48,27 +75,47 @@
...
+  display: table;
+  margin: 0 auto;
+}
+
+.topic-list {
+  list-style-type: none;
 }
 
 .create-first-target {
+  margin-left: auto;
```
Comment (vitogit) [inline app/assets/stylesheets/index.scss]: you can use short version margin: 72px auto 80px auto.   This is (top,right, bottom, left)

File: app/assets/stylesheets/index.scss [hunk 9]
```diff
@@ -48,27 +75,47 @@
...
+  margin-right: auto;
   text-align: center;
   margin-top: 72px;
   margin-bottom: 80px
 }
 
 .Psss-these-are-the {
+  margin-left: auto;
+  margin-right: auto;
```
Comment (vitogit) [inline app/assets/stylesheets/index.scss]: use margin short version if you can

File: app/assets/stylesheets/main.scss [hunk 10]
```diff
@@ -60,13 +75,19 @@ body {
...
-  color: #000000;
+  color: black;
 }
 
 .text-centered {
   text-align: center;
 }
 
+.text {
+  margin-left: auto;
```
Comment (vitogit) [inline app/assets/stylesheets/main.scss]: margin short version

File: app/assets/stylesheets/targets.scss [hunk 11]
```diff
@@ -1,48 +1,83 @@
...
-  background-color: grey;
+.button {
+  margin: 10px 50px 0px 50px;
+  background-color: black;
+  border: none;
+  color: white;
+  padding: 16px 32px;
+  text-decoration: none;
+  font-size: 16px;
+  -webkit-transition-duration: 0.4s; /* Safari */
```
Comment (vitogit) [inline app/assets/stylesheets/targets.scss:75]: what is this?

File: app/assets/stylesheets/delete_target.scss (hunk 7 above)
Comment (BelenRemedi) [reply app/assets/stylesheets/delete_target.scss:31]: The transition duration property is to determine the speed of the "hover" effect.

**Review by vitogit — APPROVED** [review]

**Review by JuanfraM — APPROVED** [review]

**Pull request merged**